### PR TITLE
feat: Adds an option to edit City/Country

### PR DESCRIPTION
**Pull request by iamsivin**
# Pull Request Template

## Description

This PR adds the ability to edit Country/City name from contact form

Fixes https://linear.app/chatwoot/issue/CW-357/add-an-option-to-edit-citycountry

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?
https://www.loom.com/share/7fd377f5e58d4a41b3992a954b104e0d

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/javascript/dashboard/routes/dashboard/conversation/contact/ContactForm.vue [hunk 1]
```diff
@@ -172,6 +206,11 @@ export default {
     },
     bio: {},
   },
+  computed: {
```
Comment (nithindavid) [inline app/javascript/dashboard/routes/dashboard/conversation/contact/ContactForm.vue:209]: This code can be removed.

Comment (iamsivin) [reply app/javascript/dashboard/routes/dashboard/conversation/contact/ContactForm.vue:209]: @nithindavid If I remove it from the computed property, the values will not render in multi-select.

Comment (muhsin-k) [reply app/javascript/dashboard/routes/dashboard/conversation/contact/ContactForm.vue:209]: @iamsivin We can initialise the value in `data`.

**Comment by muhsin-k** [comment]
@iamsivin Please change this empty text.
CleanShot 2023-04-03 at 17 14 07

**Review by muhsin-k — COMMENTED** [review]
@iamsivin Please see the comments.

**Comment by iamsivin** [comment]
> @iamsivin Please change this empty text. CleanShot 2023-04-03 at 17 14 07

@muhsin-k I think it's hardcoded. I couldn't find any prop. https://github.com/shentao/vue-multiselect/blob/59f8bbb41cc53375687dbe2cfb7369770d8d3f6f/src/Multiselect.vue#LL142C17-L142C117

**Review by muhsin-k — APPROVED** [review]

**Pull request merged**